src/rpc/index_util.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2016 BitPay, Inc.
+// Copyright (c) 2024 The Dash Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <rpc/index_util.h>
+
+#include <txmempool.h>
+#include <uint256.h>
+#include <validation.h>
+
+bool GetAddressIndex(CBlockTreeDB& block_tree_db, const uint160 addressHash, const AddressType type,
```

Comment:
could also pass `addressHash` by reference here, in `GetAddressUnspentIndex` and in txdb

Reply:
Was mistaken in thinking a `uint160` can be constructed inline in the same vein that a `uint256` can be using `uint256S`, resolved in latest push!